@xen-orchestra/fs/src/s3.js

```diff
@@ -0,0 +1,17 @@
+import { parse } from 'xo-remote-parser'
+
+import RemoteHandlerAbstract from './abstract'
+
+export default class S3Handler extends RemoteHandlerAbstract {
+  constructor(remote, opts) {
+    const { host, port, path } = parse(remote.url)
+    super(remote, opts, {
+      type: 'nfs',
+      device: `${host}${port !== undefined ? ':' + port : ''}:${path}`,
+    })
```

Comment:
What's this third args to `super`?

Reply:
it's just random crap coming from a copy/paste for now, this class will be filled in later.

Reply:
Ok.